docs/source/dev_guide/releasing-e3sm-diags.rst

```diff
@@ -161,120 +161,29 @@ This will have the ``conda/meta.yaml`` we edited in the first section.
 Releasing The Software On Anaconda
 ----------------------------------
 
-1. Since we're building with ``noarch``, you can run the below steps on
-either a Linux or macOS machine. You **do not** need to run this steps on both.
+1. The CI/CD workflow takes care of this step. Be sure to tag a release.
```

Comment:
1. Tag a release on GitHub. This triggers the CI/CD workflow that handles Anaconda releases.
2. Wait until the CI/CD build is successful.
3. ....